changelog/dmd.hexstring-cast.dd

```diff
@@ -11,10 +11,19 @@ immutable uint[] data = cast(immutable uint[]) x"AABBCCDD";
 static assert(data[0] == 0xAABBCCDD);
 ---
 
-When the hex string has a character postfix, or its length is not a multiple of the element size, it is an error:
+Character postfixes can now also be used for integers of size 2 or 4:
+---
+immutable ushort[] f = x"80 3F"w;
+static assert(f[0] == 0x803F);
 
+immutable int[] f = x"80 35 FF FD"d;
+static assert(f[0] == 0x803FFF);
```

Comment:
I don't understand the value in the assert. Shouldn't it be 0x8035FFFD ?!?

Reply:
Looks like a copy-pase error from the above line @dkorpel

Reply:
Good dcatch